docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst

```diff
@@ -10,88 +10,89 @@ While we're going to debug these issues 'manually' here, workflows (which you'll
 
 Note that besides the ``aiida-quantumespresso`` plugin, AiiDA comes with plugins for a range of other codes, all of which are listed in the `AiiDA plugin registry <https://aiidateam.github.io/aiida-registry/>`_.
 
-Set up a code
--------------
+Computer setup
+--------------
 
-To run an external code through AiiDA, such as we will be doing with Quantum ESPRESSO's pw.x in this tutorial, we first need to set it up.
-Each code needs to be associated with a computer on which it can run, therefore we first need to configure a computer.
-For the purposes of this tutorial, we are going to run the calculations on the local machine, also referred to as "localhost".
+In a production environment, AiiDA would typically be running on your work station or laptop, while launching calculations
+on some remote high-performance compute resource that you have SSH access to.
+For this reason AiiDA has the concept of a ``Computer`` to run calculations on.
 
-A computer is setup with the command ``verdi computer setup``.
-Just as you have seen when :ref:`setting up a profile<setup_verdi_quicksetup>`, this command will interactively ask you for the information it needs.
-We don't want to focus here on the details of setting up a computer, so we will use the ``--config`` option of the command.
-This allows you to configure a computer from a ``.yml`` file.
-For the virtual machine, we have already prepared this configuration file that you can :download:`download here <include/configuration/computer.yml>` and it should contain:
+To keep things simple, Quantum ESPRESSO (together with several other *ab initio* codes) has been installed directly in the
+``codes`` subfolder of your virtual machine,
+and you are going to launch your first calculations on the same computer where AiiDA is installed.
+Nevertheless, we're now going to set up this computer for launching calculations:
 
-.. literalinclude:: include/configuration/computer.yml
-    :language: yaml
-
-.. warning::
+.. code:: console
 
-    The contents of this configuration file will most likely only work for the virtual machine that comes with this tutorial.
-    If you are following this tutorial on your own AiiDA installation, you will have to adapt it yourself.
-    Instructions on how to do this are explained in :ref:`this appendix<appendix_computer_code_setup>`.
+    verdi computer setup --config computer.yml
 
-You can of course also manually copy and paste the content above in a file named ``computer.yml``.
-After you have created or downloaded the file to your local working environment, call the command:
+where ``computer.yml`` is a configuration file in the `YAML format <https://en.wikipedia.org/wiki/YAML#Syntax>`_)  that you can :download:`download here <include/configuration/computer.yml>`. This are its contents:
```

Comment:
* This is its content?

contents is better suited for things like "table of contents": https://dictionary.cambridge.org/grammar/british-grammar/content-or-contents

Reply:
agreed, done!